src/hotspot/share/opto/output.cpp

```diff
@@ -864,6 +864,30 @@ void Compile::Process_OopMap_Node(MachNode *mach, int current_offset) {
           }
           scval = sv;
         }
+      } else if (obj_node->is_SafePointScalarMerge()) {
+        SafePointScalarMergeNode* smerge = obj_node->as_SafePointScalarMerge();
+        ObjectMergeValue* mv = (ObjectMergeValue*) Compile::sv_for_node_id(objs, smerge->_idx);
```

Comment:
The description (for jdk17u PR as well) could also list the minor differences when compared to https://github.com/openjdk/jdk/pull/17562/files e.g. 1. sv_for_node_id in the former is Compile::sv_for_node_id here. 2. (void) casts are needed here because...